voxblox_planning_msgs/CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ add_definitions(-std=c++11)
 #)
 add_service_files(
 	FILES
-	YawPolicyService.srv)
+	YawPolicyService.srv
+	SmootherNameService.srv)
```

Comment:
All planning messages are usually put in [mav_comm/mav_planning_msgs](https://github.com/ethz-asl/mav_comm/tree/master/mav_planning_msgs). I would move those from here to there as well so that we have everything in one palce.